packages/katello/katello/katello-debug.sh

```diff
@@ -69,15 +69,13 @@ add_cmd "sudo -u pulp PULP_SETTINGS='/etc/pulp/settings.py' DJANGO_SETTINGS_MODU
 # Qpidd (*)
 if [ $NOGENERIC -eq 0 ]; then
   add_files /etc/qpid/*
-  add_files /etc/qpidd.conf
   add_files /etc/qpid-dispatch/qdrouterd.conf
-  add_files /var/log/qdrouterd/qdrouterd.log
 fi
 
-if [ -f /etc/pki/katello/qpid_client_striped.crt ]; then
-  add_cmd "qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671" "qpid-stat-q"
-  add_cmd "qpid-stat -u --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671" "qpid-stat-u"
-  add_cmd "qpid-stat -c --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671" "qpid-stat-c"
+if [ -f /etc/pki/katello/qpid_router_client.crt ]; then
+  add_cmd "qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_router_client.crt --ssl-key=/etc/pki/katello/qpid_router_client.key --sasl-mechanism=EXTERNAL -b amqps://$HOSTNAME@localhost:5671" "qpid-stat-q"
+  add_cmd "qpid-stat -u --ssl-certificate=/etc/pki/katello/qpid_router_client.crt --ssl-key=/etc/pki/katello/qpid_router_client.key --sasl-mechanism=EXTERNAL -b amqps://$HOSTNAME@localhost:5671" "qpid-stat-u"
+  add_cmd "qpid-stat -c --ssl-certificate=/etc/pki/katello/qpid_router_client.crt --ssl-key=/etc/pki/katello/qpid_router_client.key --sasl-mechanism=EXTERNAL -b amqps://$HOSTNAME@localhost:5671" "qpid-stat-c"
```

Comment:
I tested this and things are a bit trickier now. For this to work:

```
qpid-stat -q --ssl-certificate /etc/pki/katello/certs/pipe-katello-server-nightly-centos8.wareagle.example.com-qpid-broker.crt --ssl-key /etc/pki/katello/private/pipe-katello-server-nightly-centos8.wareagle.example.com-qpid-broker.key --sasl-mechanism=EXTERNAL -b amqps://$HOSTNAME@localhost:5671
```

Note the set of certs used are not as predictable because they are based on the hostname as well.

Reply:
@ehelms looking on a 7 box it fails with the sasl, so are you fine with me adding an if statement for the os version?

```bash
[root@katello foreman-debug-JIBCJ]# qpid-stat -q --ssl-certificate=/etc/pki/katello/certs/katello.croberts.lan-qpid-broker.crt --ssl-key=/etc/pki/katello/private/katello.croberts.lan-qpid-broker.key --sasl-mechanism=EXTERNAL -b amqps://$HOSTNAME@localhost:5671
Failed: AuthenticationFailure - Error in sasl_client_start (-4) SASL(-4): no mechanism available:
```

I think I found a way to do the hostname-cert

Reply:
Odd that it failed that way on EL7. I'd need to spin one up to try a few things. Our acceptance tests using this on EL7 with qpid-config are working. I wouldnt think qpid-stat should work differently.

Reply:
Will work on the hostname part with EL8, while you look at the EL7 part.

Reply:
If we go that route, I can just add an if statement to check the OS versions since sasl does not work with EL7 if that sounds good to you?

Reply:
If that means one command if EL7 and one if EL8, :+1:

Reply:
@ehelms with the latest update, I was able to see the commands in the foreman-debug from both EL7/8